### ISSUE TITLE
How should taxes be handled?

### ISSUE THREAD

**Issue by kevinwin** [bucko13/crypto-gps#17]
Tax rules are generally different across different jurisdictions but it'd be good to point peeps to tax rules for different jurisdictions or anything that can help them navigate the additional complexity crypto adds to taxes.

**Comment by bucko13** [comment]
Agreed! Maybe there should be a `Step-4` dedicated to Cryptocurrency Trading and in there is a section for tax links and hints?
